cfme/tests/cloud_infra_common/test_quota_with_multiple_providers.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+
+from cfme.infrastructure.provider.rhevm import RHEVMProvider
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
+from cfme.markers.env_markers.provider import ONE_PER_TYPE
+from cfme.utils.appliance.implementations.ui import navigate_to
+
+
+pytestmark = [
+    pytest.mark.provider(
+        classes=[RHEVMProvider],
+        selector=ONE_PER_TYPE,
+    ),
+    pytest.mark.provider(
+        classes=[VMwareProvider],
+        selector=ONE_PER_TYPE,
+        fixture_name='second_provider',
+    )
+]
+
+
+def test_show_quota_used_on_tenant_screen(request, appliance, v2v_providers):
+    """Test show quota used on tenant quota screen even when no quotas are set.
+
+    Steps:
+        1. Navigate to provider's 'Details' page.
+        2. Fetch the information of 'number of VMs'.
+        3. Navigate to 'Details' page of 'My Company' tenant.
+        4. Go to tenant quota table.
+        5. Check whether number of VMs are equal to number of VMs in 'in use' column.
+    """
+    view = navigate_to(v2v_providers.vmware_provider, "Details")
```

Comment:
Required:
This navigation not needed `refresh_provider_relationships_ui` method will take care of this.

Reply:
I refreshed providers with rest. It works well.